test/functional/sim_coinselect.py

```diff
@@ -125,6 +145,7 @@ def add_options(self, parser):
         parser.add_argument("--payments", default=None, required="--feerates" in sys.argv, action=PaymentsFeeratesOptionsAction)
         parser.add_argument("--feerates", default=None, required="--payments" in sys.argv, action=PaymentsFeeratesOptionsAction)
         parser.add_argument("--ops", type=int, default=None)
+        parser.add_argument("--weights", type=int, nargs='+', default=None, required="--weights" in sys.argv, action=OutputTypeWeightsOptionsAction, help="Weight values for each address type. Weights must add to 100 and be provided in the following order: bech32m bech32 p2sh-segwit legacy")
```

Comment:
Let's be a bit clearer what this does in the first place:

```suggestion
        parser.add_argument("--weights", type=int, nargs='+', default=None, required="--weights" in sys.argv, action=OutputTypeWeightsOptionsAction, help="Causes recipient output types to be chosen randomly with provided weights per address type. Weights must add to 100 and be provided in the following order: bech32m bech32 p2sh-segwit legacy")
```

Reply:
added your wording, thx!